functions.php

```diff
@@ -154,6 +154,19 @@ function wp_auth0_delete_auth0_object( $user_id ) {
 	WP_Auth0_UsersRepo::delete_meta( $user_id, 'auth0_transient_email_update' );
 }
 
+/**
```

Comment:
Can we get a better description on this function?  I'm still not entirely clear what it does.